storage/src/store/operation.rs

```diff
@@ -0,0 +1,220 @@
+//! Operations that can be applied to a [`Store`] database.
+//!
+//! The [`Operation`] enum implements the `Array` trait, allowing for a persistent log of operations
+//! based on a `crate::Journal`.
+//!
+//! [`Store`]: crate::store::Store
+
+use bytes::{Buf, BufMut};
+use commonware_codec::{Codec, EncodeSize, Error as CodecError, Read, ReadExt, Write};
+use commonware_utils::Array;
+use std::{
+    cmp::{Ord, PartialOrd},
+    fmt::{Debug, Display},
+    hash::Hash,
+};
+
+/// An operation applied to an unauthenticated database with a variable size value.
+#[derive(Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug)]
+pub enum Operation<K: Array, V: Codec> {
```

Comment:
Please provide similar descriptions for each op: https://github.com/commonwarexyz/monorepo/blob/18e6449f12f57d4e2b3ae852be6676d24e204e08/storage/src/adb/operation.rs#L38-L46

(I like set/delete/commit -> i.e. what you have for this)